docs/how-tos/nexus-validation-dev-notes.md

```diff
@@ -0,0 +1,257 @@
+# NeXus Validation Dev. Notes
+
+# 1\. Overview of validation methods
+
+nexusformat.org listed two programs for the verification and validataion of NeXus files:  
+[https://manual.nexusformat.org/validation.htm](<https://manual.nexusformat.org/validation.html>)
+
+1. nxvalidate
+
+2. punx
+
+## nxvalidate
+
+[https://github.com/nexusformat/cnxvalidate](<https://github.com/nexusformat/cnxvalidate>)
+
+### Pro:
+
+1. FAIRmat and NIAC NeXus definiton can be used.
+
+2. Easy to install with Linux
+
+3. Points out some Errors with pynxtools-ellips created file - our fault?
+
+### Con: 
+
+1. Is C/C++ based
+
+2. Not possible for Windows (cmake did not found the libxml2 library. Other libs were found after some work)
```

Comment:
Would be careful here, in general C/C++ programs can be compiled with Windows but indeed it might be trickier than for Linux, your tests showed it is non trivial but that does not mean it is not possible

Reply:
Yea, I can relax this statement.
In principle it should be possible. I would like to make this working as well on windows, but is most probably needs a detailed installation guide.
The file [nexus-validation-dev-notes.md](https://github.com/FAIRmat-NFDI/pynxtools/pull/402/files#diff-556a74c3bcde6d879691e926fde4194dead5d8dfd2b1c4614eaac91c11b8cc21) contains my succesful steps for windows - Except the last xml2 library. Maybe I can find a solution for this later.